Sources/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -220,4 +220,34 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
             completion?($0.error)
         }
     }
+    
+    /// Queries the watchers of a channel.
+    ///
+    /// For more information about channel watchers, please check [documentation](https://getstream.io/chat/docs/ios/watch_channel/?language=swift)
+    ///
+    /// - Parameters:
+    ///   - query: Query object for watchers. See `ChannelWatcherListQuery`
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    func channelWatchers(query: ChannelWatcherListQuery, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .channelWatchers(query: query)) { (result: Result<ChannelPayload<ExtraData>, Error>) in
+            do {
+                let payload = try result.get()
+                self.database.write { (session) in
+                    if query.pagination.offset == nil || query.pagination.offset == 0 {
```

Comment:
What's the semantic difference between `nil` and `0`? Maybe the offset shouldn't be optional?

Reply:
We don't encode `nil` and backend assumes 0 in that case